pandas/_libs/tslibs/period.pyx

```diff
@@ -1416,6 +1416,32 @@ cdef class _Period(object):
 
     @property
     def weekday(self):
+        """
+        Get day of the week that a Period falls on.
+
+        Returns
+        -------
+        int
+
+        See Also
+        --------
+        Period.dayofweek : Get the day component of the Period.
```

Comment:
From the description, I can't see the difference between `weekday` and `dayofweek`. Can you clarify it please?

Reply:
yup sure :+1: